cpp/ql/test/library-tests/dataflow/taint-tests/vector.cpp

```diff
@@ -325,3 +325,58 @@ void test_constructors_more() {
 	sink(v3);
 	sink(v4); // tainted
 }
+
+void taint_vector_output_iterator(std::vector<int>::iterator iter) {
```

Comment:
This doesn't seem to be used anywhere.

Reply:
I've used it in the `v4` test. There isn't flow out of the call, though.

Reply:
Thanks for fixing.